raycicmd/config.go

```diff
@@ -267,16 +281,17 @@ func ciDefaultConfig(envs Envs) *config {
 	case rayBranchPipeline, rayV2PostmergePipeline, rayCIPipeline:
 		return branchPipelineConfig
 	case rayPRPipeline, rayV2PremergePipeline, rayDevPipeline:
-		return prPipelineConfig("ray-pr", nil)
+		return prPipelineConfig("ray-pr", nil, -1)
```

Comment:
hmm.. can we use 0 value as default, and maybe use `-1` as disable (in the future). go conventionally uses zero values as default values whenever possible.